chord_metadata_service/chord/ingest/readsets.py

```diff
@@ -5,7 +5,7 @@
 # the table_id is required to fit the bento_ingest.schema.json in bento_lib
 # it can be any existing table_id which can be validated
 # the workflow only performs copying files over to the DRS
-def ingest_readset_workflow(workflow_outputs, table_id):
+def ingest_readset_workflow(workflow_outputs, dataset_id):
```

Comment:
the readset workflow should just put stuff into DRS, so it shouldn't really have an ingest endpoint - or if it does, instead of the below "Attempting" message, it should be past-tense